### DISCUSSION TITLE
I WANT TO LERAN

### DISCUSSION THREAD

**Discussion by ghost** [rapid7/metasploit-framework#14553] [General] [answered]

**Comment by CSharperMantle** [comment]
Well there are many tutorials about the question you've asked on Google and other websites. Would you please search by yourself before posting here? You question is too general to be answered.

**Comment by gwillcox-r7** [comment, marked as answer]
Closing this discussion as this seems like this wasn't intended to be a serious question anyway, and this is a place for discussion of future Metasploit features, not help requests, which should be directed to rapid7.slack.com.
